components/atomic/buttons/ButtonControl/ButtonControl.styles.ts

```diff
@@ -80,7 +84,31 @@ export const ButtonText = styled.span<Pick<BaseProps, "size" | "icon">>`
     `}
 `;
 
-export const Disclosure = styled(DialogDisclosure)`
-  --dropdown-list-item-padding: ${basePadding(2)}, ${basePadding(6)};
-  --button-background: none;
+export const Disclosure = styled(DialogDisclosure)<DisclosureProps>`
+  border: 1px solid transparent;
+  border-radius: ${pxToRem("4px")};
+
+  &:focus {
+    outline: 0;
+  }
+
+  &:focus-visible:not(:hover) {
+    background-color: var(--brand20);
+    border-color: var(--brand100);
+    ${aGlow("lightMode")}
+  }
+
+  ${({ breakpoint }) =>
+    breakpoint &&
+    respond(
+      css`
+        --dropdown-list-item-padding: 0;
+
+        > * {
+          padding: ${basePadding(2)} ${basePadding(6)};
+          background: transparent;
+        }
+      `,
+      breakpoint
+    )}
```

Comment:
It seemed to me that when `Disclosure` inherits a padding from `Dropdown` it needs to ignore it itself and pass it down to its child that's using the main `ButtonControl` styles. But maybe there's a better way?